### PR TITLE
Bugfix/bug 798106

### PR DESCRIPTION
**Pull request by baylp2022**
BUG-798106 - Unable to set speculative exec policy props and custom retry count in pega-helm-chart

### CODE REVIEW

**Review by kingakowalska1 — CHANGES_REQUESTED** [review]
Hi,

My only comment is that there should be periods at the end of the sentences in the Purpose column.

For example:

Is:
Specify the number of retries attempted when CASSANDRA_CUSTOM_RETRY_POLICY is true

Should be:
Specify the number of retries attempted when CASSANDRA_CUSTOM_RETRY_POLICY is true.

Thanks!
Kinga

**Review by taz-pega-work — COMMENTED** [review]
Can we tighten up the language and add default valuies if there are any?

File: README.md [hunk 1]
```diff
@@ -182,7 +182,10 @@ CASSANDRA_KEYSPACES_PREFIX | Specify a prefix to use when creating Pega-managed
 CASSANDRA_EXTENDED_TOKEN_AWARE_POLICY | Enable an extended token aware policy for use when a Cassandra range query runs. When enabled this policy selects a token from the token range to determine which Cassandra node to send the request. Before you can enable this policy, you must configure the token range partitioner. | `false`
 CASSANDRA_LATENCY_AWARE_POLICY | Enable a latency awareness policy, which collects the latencies of the queries for each Cassandra node and maintains a per-node latency score (an average). | `false`
 CASSANDRA_CUSTOM_RETRY_POLICY | Enable the use of a customized retry policy for your Pega Platform deployment. After enabling this policy in your deployment configuration, Cassandra queries that timeout will be retried. The number of retries may be configured using the dynamic system setting (DSS): dnode/cassandra_custom_retry_policy/retryCount. If not configured, queries will be retried once. | `false`
+CASSANDRA_CUSTOM_RETRY_POLICY_COUNT | Specify the number of retries attempted when CASSANDRA_CUSTOM_RETRY_POLICY is true |
```
[the remark this answers is not in the document]
Comment (baylp2022) [reply README.md]: Thank you. I made the recommended changes.

**Review by taz-pega-work — COMMENTED** [review]
looks good. thanks for collaborating!

**Review by taz-pega-work — APPROVED** [review]
excellent updates here. Thanks for your collaboration @baylp2022 . These read well now.

**Review by kingakowalska1 — APPROVED** [review]
Looks great, thanks!

**Comment by taz-pega-work** [comment]
with the reviewers complete, this looks ready to mearge

**Pull request merged**